zadani/soft/style.css

```diff
@@ -1,43 +1,304 @@
+                                                                                                                        color: white;
+                                                                                                                    }
+                                                                                                                }
+                                                                                                                /*desktop*/
+                                                                                                                
+                                                                                                                @media (min-width: 1000px) {
+                                                                                                                    body {
+                                                                                                                        padding: 0;
+                                                                                                                        margin: 0;
+                                                                                                                        background: url(../../images/island-hlavicka.jpg);
+                                                                                                                        background-size: cover;
+                                                                                                                        background-repeat: no-repeat;
+                                                                                                                        max-height: 500px;
+                                                                                                                    }
+                                                                                                                    .obal {
+                                                                                                                        width: 800px;
+                                                                                                                        margin: 0 auto;
+                                                                                                                        background: white;
+                                                                                                                        box-shadow: 0 0 20px rgba(0, 0, 0, 0.2);
+                                                                                                                    }
+                                                                                                                    /*nevím si rady s tím jak udělat hlavičku podle předlohy. na pozadí za .obal je hlavicka ale taky html html.. vlastně vůbec nevím jak to elegantně vyřešit.
```

Comment:
Tady hlavičku smaž, určitě nechceme mít hlavičku jako pozadí body. Chceš, aby obsah stránky byl vyjetý o kus nahoru, tak můžeš zkusit nastavit obsahu stránky (u tebe třída desktopmenu) záporný margin zeshora, např.: 
.desktopmenu {
    margin-top: -100px;
}

Přesnou hodnotu si uprav 🙂

Reply:
A uvnitř toho obsahu bys chtěla mít i patičku (ta bude mít jen jinou barvu pozadí), ať má stejnou šířku jako body a zarovnání na střed